src/test/java/com/wikia/webdriver/common/contentpatterns/MercurySubpages.java

```diff
@@ -29,6 +29,11 @@ private MercurySubpages() {}
 
   // Articles prepared for mercurycc.wikia.com wiki (with FC, CC, TA and TV)
   public static final String CC_MAIN_PAGE = "Mercury_CC_Wikia";
+  // Categories in mercurycc.wikia.com
+  public static final String CC_CATEGORY_ARTICLES = "main/category/Articles";
+  public static final String CC_CATEGORY_BLOGS = "main/category/Blog_posts";
+  public static final String CC_CATEGORY_IMAGES = "main/category/Images";
+  public static final String CC_CATEGORY_VIDEOS = "main/category/Videos";
```

Comment:
It's classification topic again 
Current solution uses wikis as core classifier for the content 
Bogna proposition is to have functionality as the core classifier 

Intuitively Bogna proposition is better in terms of being independent from tested environment
On the other hand the tested data environment is part of test composition. That's the direction we want to follow

I'd leave it for open discussion